packages/forms/src/model.ts

```diff
@@ -1227,7 +1226,139 @@ export abstract class AbstractControl {
  *
  * @publicApi
  */
-export class FormControl extends AbstractControl {
+export declare interface FormControl extends AbstractControl {
+  /** @internal */
+  _onChange: Function[];
+
+  /** @internal */
+  _pendingValue: any;
```

Comment:
Can we use more precise types here (and for the `_pendingChange` below)?

```suggestion
  _pendingValue: boolean;
```

Reply:
Done.